docs/android-feature-api.md

```diff
@@ -6,27 +6,34 @@ slug: /android-feature-api
 
 ## Manifest
 
-This file should live in Fenix and be readable by the Android Feature API:
+This file should live in Fenix and be readable by the Android Feature API.
+
+Note that:
+
+- Default values are required for Android/iOS
+- Valid `type`s are `boolean`, `string`, `int`, `drawable`
+- `choices` is optional
 
 ```yaml
 default_menu_message:
   description: "A menu message that asks people to set to default",
   variables:
     enabled:
       type: "boolean",
-      # Optional
+      # required
       defaultValue: false
     icon:
-      type: "string",
-      defaultValue: "firefox-logo"
+      type: "drawable",
+      choices:
+        - "firefox-logo", "firefox-logo-red"
```

Comment:
What does this convert to as a JSON blob? Or should this be an array of choices, with a new bullet point?

````yaml
      choices:
        - "firefox-logo"
        - "firefox-logo-red"
```